.github/workflows/artifacts.yaml

```diff
@@ -0,0 +1,35 @@
+name: "Pull Result Artifacts"
+
+on:
+  pull_request: []
+#  push:
+#    branches: main
+
+jobs:
+  auto-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: '0'
+          repository: 'buildsi/build-abi-containers'
+
+      - name: Get All Commits In Branch
+        id: merge-commits
+        uses: autamus/merge-commits@test/action
+
+      - name: Pull All Artifacts In Branch Runs
+        run: mkdir -p ./artifacts
+
+      - name: Pull All Artifacts In Branch Runs
+        uses: autamus/artifact-dl@fix/action-formatting
+        with:
+          git_token: ${{ secrets.GIT_TOKEN }}
+          input_commits: '${{ steps.merge-commits.outputs.commits }}'
```

Comment:
@vsoch Ohhh I know what's going on. Try changing the ' around `'${{ steps.merge-commits.outputs.commits }}'` to `"${{ steps.merge-commits.outputs.commits }}"`

Reply:
Will do! I think I copied this from an example somewhere, so we might want to change it there too.